websites/P/Pinterest/metadata.json

```diff
@@ -4,20 +4,34 @@
 		"name": "Bas950",
 		"id": "241278257335500811"
 	},
+	"contributors": [
+		{
+			"name": "Dark_Ville",
+			"id": "638080361179512853"
+		}
+	],
 	"service": "Pinterest",
 	"description": {
 		"en": "Discover recipes, home ideas, style inspiration and other ideas to try.",
 		"nl": "Ontdek recepten, huisideeën, stijlinspiratie en andere ideeën om uit te proberen.",
 		"vi_VN": "Khám phá các công thức, ý tưởng cải tiến nhà, cảm hứng và các ý tưởng khác."
 	},
 	"url": "www.pinterest.com",
-	"regExp": "([a-z0-9-]+[.])*pinterest([.][a-z]+)+[/]",
-	"version": "1.3.19",
+	"regExp": "([a-z0-9-]+[.])*pinterest([.][a-z]+)+",
```

Comment:
```
[/]
```
This never works since hostname doesnt contain a /

Reply:
Yerp that should be correct